aws/cloudfront/example/terragrunt.hcl

```diff
@@ -0,0 +1,12 @@
+# Configure Terragrunt to automatically store tfstate files in an S3 bucket
+remote_state {
+  backend = "s3"
+
+  config = {
+    encrypt        = true
+    bucket = get_env("terraform_state_s3bucket", "dfds-developer-test-terraform-state")
+    key            = "${path_relative_to_include()}/terraform.tfstate"
+    region         = "us-east-1"
```

Comment:
why us-east-1? I am guessing it is because of cloudfront has to be created there. Could we add a comment for that?